code/src/components/ErrorMessage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const backToStart = () => {
+    document.location.reload()
+  };
+
+const ErrorMessage = () => {
+    return (
+        <div className='error-message'>
+            <p>
+                Oops! Something went wrong. Your happy thought must be "lagom" as we say in Sweden. Between 5 and 140 characters
+                to be exact. No more, no less. 
+            </p>
+            <button
+                className='try-again-button' 
+                type='button'
+                tabIndex='0'
+                aria-pressed='false'
+                aria-label='return to start page'
+                onClick ={backToStart}
+            >
+                <img className='heart-img' 
+                     src={process.env.PUBLIC_URL + './icons/favourite.png'}
```

Comment:
I know you had troubles with implementing the .png, but good job with finding a solution to it! I am unsure of what ```process.env.PUBLIC_URL ``` actually does...?